ui/app/templates/components/app-breadcrumbs.hbs

```diff
@@ -1,13 +1,7 @@
-{{#each this.breadcrumbs as |breadcrumb index|}}
-  <li class="{{if (eq (inc index) this.breadcrumbs.length) "is-active"}}">
```

Comment:
This `is-active` class got dropped by accident, which causes the active crumb to not be highlighted:

Current:
<img width="913" alt="image" src="https://user-images.githubusercontent.com/775380/146448435-f2aeee2d-0a9d-46aa-b0b9-cda4a09d7255.png">

From PR:
<img width="898" alt="image" src="https://user-images.githubusercontent.com/775380/146448482-4e80809f-e834-458b-a1d2-65d6a88f2bcc.png">